platforms/web/public/locales/de/common.json

```diff
@@ -0,0 +1,37 @@
+{
+  "abbreviation": {
+    "hours": "Std",
+    "minutes": "Min"
+  },
+  "alert": {
+    "close": "Schließen",
+    "success": "Erfolg",
+    "title": "Ein Fehler ist aufgetreten"
+  },
+  "back": "Zurück",
+  "card_lock": "Artikel gesperrt",
+  "close_modal": "Panel schließen",
+  "confirmation_dialog": {
+    "close": "Abbrechen",
```

Comment:
There are som einconsistencies between formal and informal tone (Du vs Sie) - any preferences here?

Personally, would opt for informal.

Reply:
No problem and my pleasure! I'll update the lot of it in the near future. You canm decide which you prefer. Had a brief chat with @AntonLantukh on slack - apparently Netflix and spiegel use the formal one. So i think formal it is. 

Will update in one I got some time spare.